### ISSUE TITLE
Ignore 404 from elasticsearch-dsl

### ISSUE THREAD

**Issue by piedrahitapablo** [barseghyanartur/django-elasticsearch-dsl-drf#85]
Hi, first of all thanks for developing this library it has been useful on a project that I'm currently working on.

I have a suggestion to change the line `obj = self.document.get(id=self.kwargs[lookup_url_kwarg])` to `obj = self.document.get(id=self.kwargs[lookup_url_kwarg], ignore=404)`  in the `get_object` method of the `BaseDocumentViewSet` to avoid the `elasticsearch.exceptions.NotFoundError` when a document is not found an raise a 404 exception if it returns `None`.

I think that this also supresses the error message that is always printed, something like `GET http://elasticsearch:9200/test-payment-plans/doc/0 [status:404 request:0.002s]`, to show this message a django logger can be used.

the documentation on the ignore kwarg is here: https://elasticsearch-py.readthedocs.io/en/master/api.html#ignore

and the line to be changed is this: https://github.com/barseghyanartur/django-elasticsearch-dsl-drf/blob/master/src/django_elasticsearch_dsl_drf/viewsets.py#L208

I can make a PR when I have the time.

Again, thanks for your work.

**Comment by piedrahitapablo** [comment]
Also, if 404 is not ignored the server responds with a 500 code when a document is not found. As a temporal workaround I have to do this

```
def retrieve(self, request, pk=None, *args, **kwargs):
    try:
        return super(DocumentViewSet, self).retrieve(request, pk, *args, **kwargs)
    except ESNotFoundError:
        return Response(
            data={'detail': 'Not found.'},
            status=status.HTTP_404_NOT_FOUND,
        )
```

**Comment by barseghyanartur** [comment]
@piedrahitapablo:

Fine, but I would rather want it in combination with some flag defined on the ViewSet level. For instance, `ignore_404`.

**Comment by barseghyanartur** [comment]
@piedrahitapablo:

Solved in 0.16.3.

**Comment by piedrahitapablo** [comment]
Thank you very much!
